c/tskit/tables.c

```diff
@@ -768,13 +843,6 @@ tsk_reference_sequence_takeset_data(
     return takeset_string(&self->data, &self->data_length, data, data_length);
 }
 
-int
```

Comment:
What's the rationale for removing these - are we not using them in the ``load`` codepath and you figure it's just not worthwhile? (I don't mind, just checking)

Reply:
It was to make them consistent with the other tables where schemas and other "short" strings are don't have a `takeset`. I really didn't see the point in adding a `takeset_metadata_schema` to all the tables when I can't ever imagine it being over 1MB. (Famous last words!)

Reply:
Sure - can always add it later.